ARMTemplates/101-dtl-create-vm-builtin-user/azuredeploy.json

```diff
@@ -29,26 +29,21 @@
   },
   "variables": {
     "resourceName": "[concat(parameters('existingLabName'), '/', parameters('newVMName'))]",
-    "resourceType": "Microsoft.DevTestLab/labs/environments"
+    "resourceType": "Microsoft.DevTestLab/labs/virtualMachines"
   },
   "resources": [
     {
       "apiVersion": "2015-05-21-preview",
-      "type": "Microsoft.DevTestLab/labs/environments",
+      "type": "Microsoft.DevTestLab/labs/virtualMachines",
       "name": "[variables('resourceName')]",
       "location": "[resourceGroup().location]",
       "properties": {
-        "vms": [
-          {
-            "name": "[parameters('newVMName')]",
-            "vmTemplateName": "[parameters('existingCustomImageName')]",
-            "size": "[parameters('newVMSize')]",
-            "isAuthenticationWithSshKey": false,
-            "sshKey": "",
-            "userName": "",
-            "password": ""
-          }
-        ]
+        "vmTemplateName": "[parameters('existingCustomImageName')]",
```

Comment:
nope. the name is specific in the "name" out side the properties. basically using the same name as the DTL VM resource name.